modules/ceres-solver/2.2.0/MODULE.bazel

```
@@ -0,0 +1,10 @@
+module(
+    name = "ceres-solver",
+    version = "2.2.0",
+    compatibility_level = 2,
+)
+
+bazel_dep(name = "gflags", version = "2.2.2")
+bazel_dep(name = "glog", version = "0.7.0")
+bazel_dep(name = "eigen", version = "3.4.0")
```

I don't think we have a ready-made command for testing projects that aren't set up as Bazel modules.

I do suspect that you can fix the CI issue by changing this line (here and in the patch):

```suggestion
bazel_dep(name = "eigen", version = "3.4.0", repo_name = "com_gitlab_libeigen_eigen")
```